### PR TITLE
adding lunchbytes Google meet and jamboard links

### PR DESCRIPTION
**Pull request by frb-yousefi**

### CODE REVIEW

**Review by Robadob — APPROVED** [review]
LGTM

File: _events/lunchbytes-2021-05-12.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ R and its ecosystem of packages offers a wide variety of statistical and graphic
 In addition to its powerful analytical features, the R ecosystem provides a large number of tools and conventions to help support more open, robust and reproducible research. This includes tools for managing research projects, building robust analysis workflows, documenting data and code, testing code and disseminating and sharing analyses.
 In this talk we’ll take a whistle-stop tour of the breadth of available tools, demonstrating the ways R and the Rstudio integrated development environment can be used to underpin more open reproducible research and facilitate best practice
 
+This talk will be approximately 45 minutes, leaving 15 minutes for questions/discussion.
+
+This session will take place on [Google Meet](https://meet.google.com/dft-xnvj-fmn) and participants can join 15 minutes before the start of the session.
+
+We also have a [Google Jam Board](https://jamboard.google.com/d/1WLdKli_Lpm96tCTYHt63jLUf_xbTlubbpobphPR44Hk/edit?usp=sharing) where you can note down any questions or comments before or during the event.
```
Comment (willfurnass) [inline _events/lunchbytes-2021-05-12.md:25]: Not yet viewable/editable to folks at TUOS?

**Review by willfurnass — APPROVED** [review]

File: _events/lunchbytes-2021-05-12.md (hunk 1 above)
Comment (Robadob) [reply _events/lunchbytes-2021-05-12.md:25]: Oops, didn't think to actually check the links worked

**Comment by bobturneruk** [comment]
@willfurnass - are you proposing a change to the settings on the JamBoard?

**Comment by willfurnass** [comment]
@bobturneruk The jamboard wasn't previously editable by all at TUOS; it is now.

**Pull request merged**